src/styles/components/breadcrumbs.scss

```diff
@@ -25,6 +25,55 @@
 
 @media screen and (max-width: 1024px) {
 	.breadcrumbs {
-		display: none;
+		display: inline-block;
+		text-align: left;
+		margin: auto 0;
+		h4 {
+			font-size: 20px;
+			display: inline-block;
+			text-align: left;
+		}
+		nav {
+			display: inline-block;
+		}
+	}
+}
+
+@media screen and (max-width: 800px) {
+	.breadcrumbs {
+		position: absolute;
+		top: 21px;
```

Comment:
@Aringoaway I am not sure about those paddings and margins to some very particular size

Reply:
I am pretty sure that it will break something else on some other screen